### PR TITLE
Reformat Cloud Office CP article to reflect actual CP

### PR DESCRIPTION
**Pull request by DonNateR**
Fixes issue #1918

### CODE REVIEW

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-8ff6d1c243/how-to/).

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-d12aa6ed3c/how-to/)

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-71bff0fbe4/how-to/)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-9b2bd7f075/how-to/).

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-81ee15ed05/how-to/)

**Comment by kmsholcomb** [comment]
Hi @DonNateR. Edits attached. I tried to verify the content with my Cloud Office account, and I saw a few discrepancies. I don't know if those should be expected (because different users see different sections), but you might verify that all of the content is accurate and up-to-date (or at least the sections where I left comments). Thanks!

[Cloud Office Control Panel overview_editsKH.pdf](https://github.com/rackerlabs/rackspace-how-to/files/990598/Cloud.Office.Control.Panel.overview_editsKH.pdf)
[Cloud Office Control Panel overview_editsKH.docx](https://github.com/rackerlabs/rackspace-how-to/files/990599/Cloud.Office.Control.Panel.overview_editsKH.docx)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-c5a25d77b6/how-to/)

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-348327985e/how-to/)

File: content/exchange/cloud-office-control-panel-overview.md [hunk 1]
```diff
@@ -1,123 +1,114 @@
...
 
-<img src="{% asset_path exchange/microsoft-exchange-cloud-office-control-panel-overview/microsoft-exchange-cp-main-1.jpg %}" alt="Cloud Office Control Panel landing page" />
+Use one of the following links to log in to the Cloud Control Panel:
 
-### Rackspace Email
+[Login for new Cloud Office customers](https://cp.rackspace.com/)
+[Login for UK and MyRackspace customers](https://my.rackspace.com/).
 
-The **Rackspace Email** section provides you with options and tools such as spam filtering, group lists, aliases, and mobile devices.
+After you log in, you see sections that represent the services for which you signed up.Each section has several options and tools to choose from. As you sign up for additional services, they appear here to enable you to manage them directly from the landing page.
```
Comment (stephamon) [inline content/exchange/cloud-office-control-panel-overview.md]: Looks like there's a missing space before "Each section"

File: content/exchange/cloud-office-control-panel-overview.md [hunk 2]
```diff
@@ -1,123 +1,114 @@
...
 
-<img src="{% asset_path exchange/microsoft-exchange-cloud-office-control-panel-overview/microsoft-exchange-cp-email-2.jpg %}" alt="Cloud Office Control Panel Rackspace Email" />
+### Reseller tools
 
-Following are some of the options available in the **Rackspace Email** section of the control panel:
+The Reseller Tools section provides reseller options for Rackspace Email, such as adding new customers: managing reseller administrators, domains and sites, and gaining access to the reseller Partner Portal.
```
Comment (stephamon) [inline content/exchange/cloud-office-control-panel-overview.md]: I think that colon should be a comma.

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-bdc2bfc740/how-to/).

**Comment by stephamon** [comment]
👍

**Pull request merged**